src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -237,16 +218,14 @@ PMComplexTest >> testComplexNumberAndIntegerAreUnequalEvenIfRealPartIsEqualToInt
 	self deny: 1 + 3 i equals: 1.
 ]
 
-{ #category : #tests }
+{ #category : #'testing - arithmetic' }
 PMComplexTest >> testConversion [
```

Comment:
I don't understand why this test is called `testConversion`
It seems to be testing different arithmetical operations

I would split it into multiple tests with meaningful names (one assertion per test)
For example: `testAdditionIntegerAndComplex`, `testAdditionFloatAndComplex`, `testAdditionComplexAndNetagiveFloat`, etc.

Reply:
That's what I thought!

I would have preferred some close to what you suggested. They are testing addition to Real Numbers, Integers and Rational Numbers.

All these makes me wonder about our tests and making them more domain-driven and documentation like.